code/controllers/subsystem/non_firing/SSmapping.dm

```diff
@@ -1,3 +1,15 @@
+/// A list of station areas that are generally safe for spawning random events
+/// in. Excludes areas like the SM, holodeck, shuttles, toxins test area, AI sat, etc.
+/// "Safe" here means that things spawned here will not have disproportionately adverse
+/// effects on the round, like spawning something directly onto the supermatter.
+GLOBAL_LIST_EMPTY(event_safe_station_areas)
+
+/// A list of station areas that are used for spawning random events. Unlike
+/// event_safe_station_areas, it includes maintenance and the AI sat. Useful for
+/// events that you know are going to be chaotic, such as destinations of a
+/// tesloose.
+GLOBAL_LIST_EMPTY(event_unrestricted_station_areas)
```

Comment:
Any chance these can be controller scoped?

Reply:
Yeah, I'd agree here. Would probably be better scoped under the event subsystem?

Reply:
right now this is running once we know all the station areas have been loaded. is it safe to assume SSevent first fires after that and I can put it in SSevents.Initialize, or is it safe to assume that SSevents exists by the time the lists get filled here in SSmapping.Initialize?

Reply:
The lists will be created the moment the controller is New'd, which they all are before any of them are even initialised. They will be populated during SSmapping.Initialize(), which is currently **after** SSevents. Those will need reordering in this PR.

Reply:
Any reason they can't be left as part of SSmapping, just scoped properly? They're fundamentally lists of map objects, even though their intended use is for maps, so keeping them here makes sense to me.